nncf/quantization/advanced_parameters.py

```diff
@@ -222,13 +236,16 @@ class AdvancedAccuracyRestorerParameters:
     :param intermediate_model_dir: Path to the folder where the model, which was fully
         quantized with initial parameters, should be saved.
     :type intermediate_model_dir: Optional[str]
+    :param backup_mode: Specifies how to revert operations to their original precision.
+    :type backup_mode: BackupMode
     """
 
     max_num_iterations: int = sys.maxsize
     tune_hyperparams: bool = False
     ranking_subset_size: Optional[int] = None
     num_ranking_workers: Optional[int] = None
     intermediate_model_dir: Optional[str] = None
+    backup_mode: BackupMode = BackupMode.FP32
```

Comment:
@andrey-churkin @KodiaqQ What do you think about this proposal?
```
mode: RestoreMode = RestoreMode.ACTIVATIONS_AND_WEIGHTS


class RestoreMode(Enum):
    ACTIVATIONS_AND_WEIGHTS = "activations_and_weights"
    ONLY_ACTIVATIONS = "only_activations"
```
Using QuantizerGroup is more flexible and is not introduced new class but give user a chance to remove FQ from weights:

```
groups: List[QuantizerGroup] = [QuantizerGroup.ACTIVATIONS, QuantizerGroup.WEIGHTS]
```


cc` @AlexKoff88

Reply:
LGTM.

Reply:
Done